packages/mermaid/src/docs/.vitepress/scripts/fetch-avatars.ts

```diff
@@ -0,0 +1,41 @@
+import fs from 'fs-extra';
+import { fileURLToPath } from 'url';
+import { $fetch } from 'ohmyfetch';
+
+const pathContributors = new URL('../contributor-names.json', import.meta.url);
+const getAvatarPath = (name: string) =>
+  new URL(`../../public/user-avatars/${name}.png`, import.meta.url);
+// const dirSponsors = resolve(docsDir, 'public/sponsors/')
+
+let contributors: string[] = [];
+
+async function download(url: string, fileName: URL) {
+  if (fs.existsSync(fileName)) return;
+  // eslint-disable-next-line no-console
+  console.log('downloading', fileName);
+  try {
+    const image = await $fetch(url, { responseType: 'arrayBuffer' });
+    await fs.writeFile(fileName, Buffer.from(image));
+  } catch {}
+}
+
+async function fetchAvatars() {
+  await fs.ensureDir(fileURLToPath(new URL('..', getAvatarPath('none'))));
```

Comment:
Btw, since Node.JS v10.12.0, Node's builtin `mkdir` function now supports the `recursive: true` option, that should do the same thing as `ensureDir`.

This means we should be able to remove the `fs-extra` library, and just use `node:fs/promises`!

See https://nodejs.org/api/fs.html#fspromisesmkdirpath-options

```suggestion
  await fs.mkdir(fileURLToPath(new URL('..', getAvatarPath('none'))), {recursive: true});
```